pom.xml

```diff
@@ -1326,6 +1328,12 @@
                     <artifactId>provisio-maven-plugin</artifactId>
                     <version>1.0.9</version>
                 </plugin>
+
+                <plugin>
```

Comment:
Pulled out to ensure common versions are used everywhere (`presto-server-rpm` and `presto-cli-rpm`).